astropy/modeling/tests/test_models.py

```diff
@@ -23,14 +23,18 @@
 from ..core import FittableModel
 from ..polynomial import PolynomialBase
 from ...tests.helper import pytest
-
+from ...utils import minversion
 
 try:
+    import scipy
     from scipy import optimize  # pylint: disable=W0611
     HAS_SCIPY = True
 except ImportError:
     HAS_SCIPY = False
 
+if HAS_SCIPY and not minversion(scipy, "0.14"):
+    HAS_SCIPY = False
```

Comment:
Just an observation: This skips all the other existing tests that currently requires SciPy but not necessarily v0.14 or later. Perhaps it is more accurate to make a new flag called `HAS_SCIPY_14` (or something like that) and then explicitly use it for `Tabular` tests only?